File: Beginner/Area or Perimeter/areaOrPeri.java
/* package codechef; // don't place package name! */
import java.util.*;


/* Name of the class has to be "Main" only if the class is public. */
class Main {
	public static void main (String[] args) {
		// your code goes here
		Scanner sc = new Scanner(System.in);
		int a, b;
		a = sc.nextInt();
		b = sc.nextInt();
		int area=a*b, peri=2*(a+b);
		if (area > peri) {
		    System.out.println("Area");
		    System.out.print(area);
		} else if (area < peri) {
		    System.out.println("Peri");
		    System.out.print(peri);
		} else {
		    System.out.println("Eq");
		    System.out.print(area);
		}
	}
}

File: Beginner/Area or Perimeter/areaOrPeri.c
#include <stdio.h>

int main(void) {
	// your code goes here
	int a, b;
	scanf("%d\n%d", &a, &b);
	int area=a*b, peri=2*(a+b);
	if (area > peri) {
	    printf("Area\n");
	    printf("%d", area);
	} else if (area < peri) {
	    printf("Peri\n");
	    printf("%d", peri);
	} else {
	    printf("Eq\n");
	    printf("%d", area);
	}
}

